Server/app.js

```diff
@@ -3,10 +3,11 @@ const bodyParser = require('body-parser');
 const config = require('./config/default');
 const DataLayerFactory = require('./datalayer/factory.datalayer');
 const routes = require('./config/routes');
+const cors = require('cors');
 
 const app = express();
 app.use(bodyParser.json({limit: '1mb'}));
-
+app.use(cors());
```

Comment:
Refer feedback on https://github.com/shashank-compro/time-management-system/pull/3/files#r325636897

This is a common change. Please coordinate with @shashank-compro

Reply:
Removed for now, will take a pull